addons/sms/views/sms_template_views.xml

```diff
@@ -52,18 +52,6 @@
                                 <field name="body" widget="sms_widget" nolabel="1"/>
                             </group>
                         </page>
-                        <page string="Dynamic Placeholder Generator" name="dynamic_placeholder_generator" groups="base.group_no_one">
```

Comment:
As SMS templates have text bodies that do not use the editor (at least last time I played with it ^^), how are people supposed to generate inline qweb syntax ?